plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/table/HudiTableFileSystemView.java

```diff
@@ -222,7 +222,7 @@ private void resetFileGroupsReplaced(HudiTimeline timeline)
                                         Map.entry(new HudiFileGroupId(entry.getKey(), fileId), instant)));
                     }
                     catch (IOException e) {
-                        throw new TrinoException(HUDI_BAD_DATA, "error reading commit metadata for " + instant);
+                        throw new TrinoException(HUDI_BAD_DATA, "error reading commit metadata for " + instant, e);
```

Comment:
This change looks unrelated to "Fix replace commit handling in Hudi Active Timeline". Please separate a commit.

Reply:
Removed this small change, will prepare a very simple commit for this in the future.